### ISSUE TITLE
Missing input in `MergeRequestDraftNotes.create`

### ISSUE THREAD

**Issue by NatoBoram** [jdalrymple/gitbeaker#3398]
**Description**

I'm trying to submit draft notes via the API but it seems like there's a few missing fields in the API.

For example:

* https://docs.gitlab.com/ee/api/draft_notes.html#create-a-draft-note

In the web UI, it sends something like that:

```json
{
	"view": "parallel",
	"line_type": "old",
	"merge_request_diff_head_sha": "f7a4fc34fe6f63ab2cd6b91cc12ddb83ae8d410a",
	"in_reply_to_discussion_id": "",
	"note_project_id": "",
	"target_type": "merge_request",
	"target_id": 160353401,
	"return_discussion": true,
	"draft_note": {
		"note": "New discussion on add silk touch",
		"position": "{\"base_sha\":\"997a91b00cd3974ab4aae8275be85e98c7656875\",\"start_sha\":\"997a91b00cd3974ab4aae8275be85e98c7656875\",\"head_sha\":\"f7a4fc34fe6f63ab2cd6b91cc12ddb83ae8d410a\",\"old_path\":\"src/main/java/com/natoboram/switcheroo/BlockSwitch.java\",\"new_path\":\"src/main/java/com/natoboram/switcheroo/BlockSwitch.java\",\"position_type\":\"text\",\"old_line\":null,\"new_line\":114,\"line_range\":{\"start\":{\"line_code\":\"a044cf87d56aaa49cfa84e4c22f043315ca3c88a_109_110\",\"type\":\"new\",\"old_line\":null,\"new_line\":110},\"end\":{\"line_code\":\"a044cf87d56aaa49cfa84e4c22f043315ca3c88a_109_114\",\"type\":\"new\",\"old_line\":null,\"new_line\":114}},\"ignore_whitespace_change\":true}",
		"noteable_type": "MergeRequest",
		"noteable_id": 160353401,
		"commit_id": null,
		"type": "DiffNote",
		"line_code": "a044cf87d56aaa49cfa84e4c22f043315ca3c88a_109_114"
	}
}
```

With this package, I only have these options:

```ts
const draft = await this.#client.MergeRequestDraftNotes.create(
	this.#event.object_attributes.target_project_id,
	this.#event.object_attributes.iid,
	comment.body,
	{
		commitId: comment.sha ?? undefined,
		inReplyToDiscussionId: comment.parentId ?? undefined,
		showExpanded: false,
		resolveDiscussion: false,
	},
)
```

Am I calling the correct API?

Is there a way to send a `position`?

**Possible solutions**

Add missing fields to `MergeRequestDraftNotes.create`

**Checklist**
- [x] I have checked that this is not a duplicate issue.
- [x] I have read the documentation.

**Comment by jdalrymple** [comment]
I just missed them from the type! Easy fix!

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`39.13.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/39.13.0) :rocket:
